Library/SharedFunctions.swift

```diff
@@ -188,8 +188,33 @@ public func updatedUserWithClearedActivityCountProducer() -> SignalProducer<User
     .demoteErrors()
 }
 
-internal func classNameWithoutModule(_ klass: AnyClass) -> String {
-  return klass
+public func formattedAmountForRewardOrBacking(
```

Comment:
I'm not sure this should be a shared function, it seems pretty specific to the reward view 🤔

Reply:
Oh I see now why it's used in two places. I'm still not sure this `SharedFunctions` file is the right  place for it though 🤔 🤔

Reply:
Suggestions?